### DISCUSSION TITLE
POW camp

### DISCUSSION THREAD

**Discussion by Vsauce07** [Vdauphin/HeartsAndMinds#1325] [Question and answer] [answered]
**Hello,**
is it possible to make a POW camp where if you put prisoners, it increases the reputation based on their faction? It's a bit odd that after you arrest someone, you have to kill or leave them there. Thanks in advance!

**Comment by Vdauphin** [comment, marked as answer]
Hello,

>  It's a bit odd that after you arrest someone, you have to kill or leave them there. 

See http://vdauphin.github.io/HeartsAndMinds/InGame-documentation#prisoners

Cheers

**Reply by ghost** [reply]
I wonder if we could create a trigger that fires once every x minutes and makes POWs play sit animation, after x amount of time it trades them in for respawn tickets?

**Reply by Vdauphin** [reply]
Hello,
You can do whatever you want
Cheers
